docs/tutorials/elasticdl_local.md

```diff
@@ -99,22 +99,51 @@ retrieve the source code, please run the following command.
 git clone https://github.com/sql-machine-learning/elasticdl
 ```
 
-The model definitions are in directory `elasticdl/model_zoo`.  The following
-commands build the Docker image `elasticdl:mnist`
+The model definitions are in directory `elasticdl/model_zoo`.
+
+### Build the Model Zoo for Parameter Server
+
+The following commands build the Docker image `elasticdl:mnist_ps`
+
+```bash
+cd elasticdl
+elasticdl zoo init --model_zoo=model_zoo
+elasticdl zoo build --image=elasticdl:mnist_ps .
+```
+
+### Build the Model Zoo for AllReduce
+
+We must build the base image `elasticdl:dev_allreduce` firstly using the
+
+```bash
+scripts/travis/build_images.sh
+```
+
+We have not released ElasticDL packages with AllRedue yet. So we need to
+build packages manually by the command.
+
+```bash
+scripts/docker_build_wheel.sh
+```
+
+Then, we can build the AllReduce training image `elasticdl:mnist_allreduce`
+with model definitions in `model_zoo`.
 
 ```bash
-cd elasticdl/model_zoo
-elasticdl zoo init
-elasticdl zoo build --image=elasticdl:mnist .
+elasticdl zoo init \
+  --base_image=elasticdl:dev_allreduce \
```

Comment:
Can we ship a base image for AllReduce job at this time?
After horovod release the pip package, we can use the pip install way instead.

Reply:
After AllReduce is stable, we can ship a base image. After Horovod releases its latest package, we can add it to the requirement of ElasticDL.

Reply:
We can upload a runnable image at this time. After it's stable, we can overwrite the image with the stable version.

Reply:
It is slow to upload the image. I will use the AllReduce base image in the next PR after I succeed in shipping the image.